content/en/real_user_monitoring/reactnative/troubleshooting.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Troubleshooting
+kind: documentation
+
+---
+
+If you experience unexpected behavior with Datadog React Native RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog Support][1] for further assistance.
+
+## No data is being sent to Datadog
+
+Follow these instructions when the SDK has been installed and the app compiles, but no data is received by Datadog. We recommend following these steps in order.
+
+### Check the configuration
+
+Sometimes, no data gets sent due to a small misstep in the configuration. Here are common things to check:
+
+- Ensure your `clientToken` and `applicationId` are correct
+- Make sure you have not set `sessionSamplingRate` to something other than 100 (100 is the default value), or else your session might not send
+- If you've set up a `Proxy` in the Datadog configuration, check that it has been correctly configured
+- Check that you are **tracking views** (all events must be attached to a view) and **sending events**
+
+### Review SDK logs in React Native
+
+- Set `config.verbosity = SdkVerbosity.DEBUG` (import `SdkVerbosity` from `@datadog/mobile-react-native`)
+- Logs will appear in the Javascript console, you should expect to see an output like this one:
+
+  ```
+  INFO  DATADOG: Datadog SDK was initialized
+  INFO  DATADOG: Datadog SDK is tracking interactions
+  INFO  DATADOG: Datadog SDK is tracking XHR resources
+  INFO  DATADOG: Datadog SDK is tracking errors
+  DEBUG  DATADOG: Starting RUM View “Products” #Products-oaZlP_FVwGM5vtPoup_rT
+  DEBUG  DATADOG: Adding RUM Action “RCTView” (TAP)
+  ```
+
+  In this example, the first 4 logs indicate that the SDK has been correctly configured and the last 2 lines are events that were sent.
+
+#### Possible cause
+
+If you are on iOS and see some DEBUG logs indicating that logs or RUM events were sent **before** the initialization logs, it might be the reason why the SDK is not sending events.
```

Comment:
```suggestion
If you are on iOS and see some DEBUG logs indicating that logs or RUM events were sent **before** the initialization logs, this may be why the SDK is not sending events.
```